liquibase-standard/src/main/java/liquibase/database/Database.java

```diff
@@ -655,6 +655,14 @@ default boolean supportsCreateIfNotExists(Class<? extends DatabaseObject> type)
         return false;
     }
 
+    /**
+     * Does the database support the "ROWDEPENDENCIES" syntax for table creation?
+     * @return true if the "ROWDEPENDENCIES" syntax is supported for table creation, false otherwise.
+     */
+    default boolean supportsCreateRowDependencies() {
```

Comment:
I guess we don't need this new supports method as this is only supported by Oracle and this class already has plenty of `supports` methods.  Just adding `instanceof OracleDatabase` where it would be called is enough.

Reply:
Done.